### PR TITLE
Add Support for Keypad Button Presses

### PR DESCRIPTION
**Pull request by bdunks**
Add a method to send keypad button press commands
to the Main Repeater to trigger the buttons action

### CODE REVIEW

**Comment by bdunks** [comment]
You can see a usage of this at my initial implementation of a Lutron RadioRA2 component for HASS.

[Scene Component](https://github.com/bdunks/home-assistant/blob/lutron-scene/homeassistant/components/scene/lutron.py)
[Update to Lutron Hub](https://github.com/bdunks/home-assistant/blob/lutron-scene/homeassistant/components/lutron.py) (temporarily using forked pylutron library)

File: pylutron/__init__.py [hunk 1]
```diff
@@ -561,6 +562,11 @@ def add_button(self, button):
     dispatch button events."""
     self._buttons.append(button)
 
+  def press(self, button):
```
Comment (thecynic) [inline pylutron/__init__.py]: I wonder if this should be triggered via the button object itself? The fact that you are reaching into the button private for _num seems to tell me that we should have a cleaner interface. I never fully flushed out the keypad <> button relationship as I was just trying to parse most things.

Though, if we do that, we'll need to provide our integration id to the button, or maybe add something like button->set_keypad(...) or set_owner, something like that.

Then we'd just be able to button->trigger_press() or maybe button->toggle() or something like that.

Comment (thecynic) [reply pylutron/__init__.py]: We can also change the parser to just hand the button our keypad object at button creation.

**Comment by bdunks** [comment]
I'm going to go ahead and close this, since the intent has been fully satisfied with 0.2.0+

**Pull request closed without merging**